apps/vr-tests/src/stories/PeoplePicker/PeoplePickerNormalWithText.stories.tsx

```diff
@@ -13,7 +13,7 @@ export default {
       new Steps()
         .snapshot('default', { cropTo: '.testWrapper' })
         .setValue('.ms-BasePicker-input', 'a')
-        .snapshot('suggestion: "a"')
```

Comment:
Storywright generates a screenshot with `suggestion: "a"` in the file name, which causes `vr-approval-cli` to malfunction when processing special characters in file names.